scripts/modules/cli.py

```diff
@@ -689,7 +689,7 @@ def upload_sourcemaps_handler(self):
             '-F service_name="{service_name}" '
             '-F service_version="{service_version}" '
             '-F bundle_filepath="{bundle_path}" '
-            '-F sourcemap=@/tmp/sourcemap '
+            '-F sourcemap=@{sourcemap_file} '
```

Comment:
What's the different between `@{filename}`  and  `@filename` syntax?

Reply:
This is a Python-ism. The curly-braces are indicating interpolation from the `format()` call that is chained to the end of the string. So, the value found in L699 and substituting it for `{sourcemap_file}`.

Reply:
Forget about what I asked, for some reason I though it was bash-ish :( Thanks for the clarification :)